internal/jobqueue/dbjobqueue/dbjobqueue.go

```diff
@@ -56,6 +56,11 @@ const (
 		)
 		RETURNING token, type, args, queued_at, started_at`
 
+	sqlRequeue = `
+		UPDATE jobs
+		UNSET (started_at, token)
```

Comment:
Hmm, the `dbjobqueue` tests fail on this statement and I wasn't able to find anything about `UPDATE UNSET`. Are you sure that this is valid?

Reply:
`UPDATE SET` with token = uuid.Nil

Reply:
Updated :)